kolibri/utils/tests/test_version.py

```diff
@@ -32,46 +28,56 @@ def test_version(self):
 
     @mock.patch("kolibri.utils.version.get_git_describe", return_value=None)
     @mock.patch("kolibri.utils.version.get_version_file", return_value=None)
-    def test_alpha_0_version(self, file_mock, describe_mock):
+    @mock.patch("kolibri.utils.version.get_git_changeset", return_value=None)
+    def test_no_tag_no_file_version_no_git_changeset(
+        self, get_git_changeset_mock, file_mock, describe_mock
+    ):
         """
         Test that when doing something with a 0th alpha doesn't provoke any
         hickups with ``git describe --tag``.
+        If the version file returns nothing, and get_git_describe returns nothing,
+        then get_prerelease_version should return 0.15.8.dev0
```

Comment:
A bit of a nitpick, but I think this should be more general, rather than a specific version. i.e. explaining - what is it exactly that 0.15.8 is an example of? where is this value coming from?